plugins/inputs/openweathermap/README.md

```diff
@@ -0,0 +1,145 @@
+# Telegraf Plugin: openweathermap
+
+OpenWeatherMap provides the current weather and forecasts for more than 200,000 cities. To use this plugin you will need a token. For more information [click here](https://openweathermap.org/appid).
+
+Find city identifiers in this [list](http://bulk.openweathermap.org/sample/city.list.json.gz)
+
+### Configuration:
+
+```
+[[inputs.openweathermap]]
+  ## Root url of API to pull stats
+  base_url = "http://api.openweathermap.org/data/2.5/"
+  # Your personal user token from openweathermap.org
+  app_id = "xxxxxxxxxxxxxxxxxxxxxxx"
+  # List of city identifiers
+  cities = ["2988507", "519188"]
+  # HTTP response timeout (default: 5s)
+  response_timeout = "5s"
```

Comment:
could you also add an option for units? i noticed a little later on you were doing a conversion from Kelvin to Celsius: https://openweathermap.org/current#data

Reply:
I'm not sure, what do you have in mind? @russorat is there a new option in telegraf to declare "Kelvin" anywhere?

Reply:
Do you mean adding one config option that will control the `units=metric`url parameter?

Reply:
sorry i wasn't clear. yeah i mean the option in the config to control the metrics returned from the api. So something that controls `units=metric` in the call to the API.